### PR TITLE
Merge MONDO:0022869 into MONDO:0001389 congenital coronary artery anomaly/malformation

### PR DESCRIPTION
**Pull request by twhetzel**
closes https://github.com/monarch-initiative/mondo/issues/7607

### CODE REVIEW

**Comment by twhetzel** [comment]
@sabrinatoro I can circle back and sort out the proxy merge issue.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -350706,7 +350712,7 @@ xref: ICD10CM:Q24.5 {source="Orphanet:1081/e", source="Orphanet:1081"}
 xref: icd11.foundation:902783759 {source="MONDO:equivalentTo", source="Orphanet:1081", source="https://orcid.org/0000-0002-4142-7153"}
 xref: MedDRA:10061060 {source="Orphanet:1081/e", source="Orphanet:1081"}
 xref: MEDGEN:1612789 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: Orphanet:1081 {source="MONDO:equivalentTo"}
+xref: Orphanet:1081 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:350715]: @sabrinatoro I thought adding `source="MONDO:preferredExternal"` resolved the proxy merge. What am I forgetting 🤔

Comment (twhetzel) [reply src/ontology/mondo-edit.obo:350715]: To summarize the chat, I will remove the new xref to Orphanet:1081 on MONDO:0001389 and also remove the MONDO:preferredExternal I added on MONDO:0001390. I will also add a new ticket to merge MONDO:0001389 with MONDO:0001390.

Comment (twhetzel) [reply src/ontology/mondo-edit.obo]: New ticket [added](https://github.com/monarch-initiative/mondo/issues/8360).

**Review by sabrinatoro — APPROVED** [review]
@twhetzel I approved this PR. 
Since it is still assigned to you, I want to make sure that you are done making changes before merging.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo]: And those changes mentioned above were done.

**Comment by twhetzel** [comment]
@sabrinatoro yes, I made the changes we talked about yesterday.

**Pull request merged**